src/mono/mono/mini/mini-runtime.c

```diff
@@ -4290,8 +4290,12 @@ mini_init (const char *filename, const char *runtime_version)
 
 	mono_interp_stub_init ();
 #ifndef DISABLE_INTERPRETER
-	if (mono_use_interpreter)
+	if (mono_use_interpreter) {
+		if (g_hasenv ("MONO_METADATA_UPDATE") && mono_interp_opts_string == NULL) {
+			mono_interp_opts_string = "-inline";
```

Comment:
Isn't there a global variable which controls this instead of putting a getenv here ?

Reply:
No. the global variable is whether there's been any changes.  this is setting up startup.